chain/network/src/routing/routing.rs

```diff
@@ -65,7 +65,95 @@ pub enum EdgeType {
     Removed,
 }
 
-pub type Edge = Arc<EdgeInner>;
+#[derive(BorshSerialize, BorshDeserialize, Clone, Debug, PartialEq, Eq)]
+#[cfg_attr(feature = "test_features", derive(Serialize, Deserialize))]
+pub struct Edge(pub Arc<EdgeInner>);
+
+impl Edge {
```

Comment:
add comment to explain what is an edge (and that this is an undirected connection)

Reply:
Ok, I'll submit a separate PR for commenting this.

Reply:
I'll add comments in one PR after https://github.com/near/nearcore/pull/5267